docs/APIReference-APIMigration.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: v0-10-api-migration
+id: v0.10 API Migration
```

Comment:
Hmm, if we change the ID here won't we need to change all the places we reference it? Does this change the URL too? Maybe it's a good idea to leave it as is.